torch/_inductor/compile_worker/subproc_pool.py

```diff
@@ -314,5 +322,5 @@ class TestException(RuntimeError):
     pass
 
 
-def raise_testexc():
+def raise_testexc() -> None:
```

Comment:
```suggestion
def raise_testexc() -> typing_extensions.Never:
```
None is technically wrong since it can never return.